File: Java/Tiket.java
public class Tiket {
    private String id;
    private String namaFilm;
    private String jenisTiket;
    private int harga;

    public Tiket(String id, String namaFilm, String jenisTiket, int harga) {
        this.id = id; this.namaFilm = namaFilm; this.jenisTiket = jenisTiket; this.harga = harga;
    }
    public String getId() { return id; }
    public String getNamaFilm() { return namaFilm; }
    public String getJenisTiket() { return jenisTiket; }
    public int getHarga() { return harga; }
    public void setNamaFilm(String value) { namaFilm = value; }
    public void setJenisTiket(String value) { jenisTiket = value; }
    public void setHarga(int value) { harga = value; }
}

File: PHP/Tiket.php
<?php

class Tiket
{
    private $id;
    private $namaFilm;
    private $jenisTiket;
    private $harga;
    private $gambar;

    public function __construct($id, $namaFilm, $jenisTiket, $harga, $gambar)
    {
        $this->id = $id;
        $this->namaFilm = $namaFilm;
        $this->jenisTiket = $jenisTiket;
        $this->harga = $harga;
        $this->gambar = $gambar;
    }

    public function getId()
    {
        return $this->id;
    }

    public function getNamaFilm()
    {
        return $this->namaFilm;
    }

    public function getJenisTiket()
    {
        return $this->jenisTiket;
    }

    public function getHarga()
    {
        return $this->harga;
    }

    public function getGambar()
    {
        return $this->gambar;
    }

    public function setNamaFilm($namaFilm)
    {
        $this->namaFilm = $namaFilm;
    }

    public function setJenisTiket($jenisTiket)
    {
        $this->jenisTiket = $jenisTiket;
    }

    public function setHarga($harga)
    {
        $this->harga = $harga;
    }

    public function setGambar($gambar)
    {
        $this->gambar = $gambar;
    }
}
?>